src/community/wfs3/src/test/java/org/geoserver/wfs3/TilingSchemesTest.java

```diff
@@ -24,4 +25,43 @@ public void testTilingSchemesResponse() throws Exception {
         assertTrue(schemesSet.contains("GoogleMapsCompatible"));
         assertTrue(schemesSet.size() == 2);
     }
+
+    @Test
```

Comment:
Would be nice to have a test checking the error message in case an unknown tiling scheme has been requested

Reply:
Added